src/themes/index.story.css

```diff
@@ -0,0 +1,29 @@
+z-body {
+  font-size: 16px;
+  letter-spacing: 0.3px;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  width: 400px;
+  padding: 0;
+}
+
+ul > li {
+  display: flex;
+  margin: 4px;
+  justify-content: space-between;
```

Comment:
maybe it's easier to look with a border between the list items. I would add
```css
align-items: center;
border-bottom: 1px solid var(--gray200);
```

and 

```css
padding: 8px 0;
```
instead of `margin`

Reply:
ok